interactors/addressNonceHandler.go

```diff
@@ -101,6 +102,14 @@ func (anh *addressNonceHandler) sendTransaction(ctx context.Context, tx *data.Tr
 	return anh.proxy.SendTransaction(ctx, tx)
 }
 
+func (anh *addressNonceHandler) isTxAlreadySent(tx *data.Transaction) bool {
+	oldTx, nonceAlreadyUsed := anh.transactions[tx.Nonce]
```

Comment:
is this correct? After you call the "GetNonce" function you will always get a new, incremented value. 
nonceAlreadyUsed will always return false

Reply:
done